packages/aura-language-server/src/tern-server/tern-server.ts

```diff
@@ -239,6 +154,22 @@ function fileToUri(file: string): string {
     }
 }
 
+function uriToFile(uri: string): string {
+    const parsedUri = URI.parse(uri);
+    if (parsedUri.scheme) {
+        return parsedUri.fsPath;
+    }
+
+    // Handle relative path strings from tests
+    if (path.isAbsolute(uri)) {
+        return uri;
+    }
+
+    // For relative paths, we need to return the path as Tern expects it
+    // The test provides workspace-relative paths, so return them as-is
+    return uri;
```

Comment:
I don't think this is necessary, but it preserver the behavior in the tests (who are using paths relative to test-workspace).

would appreciate careful review if anyone knows how this really works.